Lib/test/test_dataclasses.py

```diff
@@ -1737,23 +1737,25 @@ class C:
                 i: int = field(metadata=0)
 
         # Make sure an empty dict works.
+        d = {}
```

Comment:
```suggestion
         # See issue 35960
        d = {} 
```

Reply:
Thanks, I have addressed this in bec2bad.